src/openxr/OpenXRApi.cpp

```diff
@@ -1546,6 +1635,7 @@ bool OpenXRApi::initialiseInstance() {
 
 	// If we have these, we use them, if not we skip related logic..
 	request_extensions[XR_MND_BALL_ON_STICK_EXTENSION_NAME] = &monado_stick_on_ball_ext;
+	request_extensions[XR_EXT_PALM_POSE_EXTENSION_NAME] = &palm_pose_ext;
```

Comment:
Shouldn't this use our default extension mechanism?

Reply:
There aren't any functions or anything so it feels like overkill. But otherwise I would say yes. Worth a discussion

Reply:
Hmm, thinking some more about this and talking to Rune about the fact that we should only supply bindings that are supported, I think there are a few enhancements we can do here to pretty this up instead of hardcode it. So I will change this.